sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -0,0 +1,195 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._generated.models import QueueDescription as InternalQueueDescription
+
+
+class QueueDescription(object):  # pylint:disable=too-many-instance-attributes
```

Comment:
Is there a reason we're using the name `QueueDescription`?
To be a bit more consistent with our SDKs it should probably be `QueueProperties` or `ServiceBusQueueProperties`.

Reply:
Also - for a subsequent release (do not bother for this preview) - we can makes these dict-like.
See BlobProperties as an example (using DictMixin).

Reply:
We changed it to QueueProperties. But the service team is against it so we changed it back for this release. Might have more discussions later.
See [more details here](https://microsoft.sharepoint.com/:o:/r/teams/AzureDeveloperExperience/_layouts/15/Doc.aspx?sourcedoc=%7B41b0af1e-8da3-4d88-b296-9b616c3e7b6b%7D&action=edit&wd=target(Design.one%7C14d18aa9-3941-cf4d-b2fa-4b211ac30733%2FProposed%20changes%20to%20Management%20APIs%7C31359120-f6d9-7d4c-8964-b71564e46614%2F))